spec/cspec/c/gen-config-thy.py

```diff
@@ -278,13 +300,14 @@ def write_config_thy(config_thy_path, config: Dict[str, str]):
                 pass
 
         f.write('\n(* These definitions should only be unfolded consciously and carefully: *)\n')
-
+        if physBase:
+            f.write('hide_fact (open) physBase_def\n')
```

Comment:
would it make sense to add physBase to names instead?

Reply:
Was planning to keep them separate for now, because I want to overhaul the config generation with the new json input. Eventually adding them to `names` (or whatever that will be then) here would make sense, yes.

Reply:
Actually, I'm generating `names` in this function, it's not from the config file. So I'll add your suggestion now already.